lib/pengine/unpack.c

```diff
@@ -3665,10 +3666,12 @@ remap_operation(struct action_history *history,
                 || (history->expected_exit_status == history->exit_status)
                 || !pcmk_is_set(history->rsc->flags, pe_rsc_managed)) {
 
+                /* For probes, recurring monitors for the Stopped role, and
```

Comment:
Yes, you can configure a recurring monitor for target-role=Stopped, and it will run on all nodes where the resource isn't supposed to be running. Sort of a paranoid option to make sure nothing gets started manually or enabled at boot, etc.